src/main/resources/view/SummaryWindow.fxml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<?import javafx.scene.Scene?>
+<?import javafx.scene.chart.PieChart?>
+<?import javafx.scene.control.Label?>
+<?import javafx.scene.image.Image?>
+<?import javafx.scene.layout.VBox?>
+<?import javafx.scene.text.Font?>
+<?import javafx.stage.Stage?>
+
+<fx:root minHeight="500.0" minWidth="500.0" resizable="false" title="Your UniCa$h Summary" type="javafx.stage.Stage" xmlns="http://javafx.com/javafx/20.0.1" xmlns:fx="http://javafx.com/fxml/1">
+   <icons>
+      <Image url="@/images/help_icon.png" />
+   </icons>
+   <scene>
+      <Scene>
+         <VBox fx:id="summaryMessageContainer" alignment="CENTER" prefHeight="472.0" prefWidth="508.0">
+            <children>
+               <Label fx:id="summaryMessage" text="Label" textAlignment="CENTER" underline="true" wrapText="true">
+                  <font>
+                     <Font name="Arial Bold" size="18.0" />
```

Comment:
We've loaded the 'Inter' font to the app at the point of initialization, so we don't need to specify! Another thing to consider is that the styles are consolidated in `DarkTheme.css` so you can consider using that if you want to add more flexible customization

Reply:
Alright, will fix this! This is just a cosmetic fix to standardize the look of all the UI components of the app is that correct?

Reply:
Yup, it's quite a trivial change that we can push to v1.3b. We can finalize the UI font element and create a header in the  css file, perhaps even incorporate it into `StyleSheet.java` so we can propagate a consistent theme across our app.